app/javascript/components/users/SignInPage.stories.tsx

```diff
@@ -31,7 +31,12 @@ export default {
 function SWRWrapper(props:React.PropsWithChildren<unknown>) {
 	return <SWRConfig value={
 		{
-			dedupingInterval: 2500, // we need to make SWR not dedupe
+			dedupingInterval: 0, // we need to make SWR not dedupe
+			revalidateOnMount: true,
+			revalidateOnFocus: true,
+			revalidateOnReconnect: true,
+			focusThrottleInterval: 0,
+			provider: () => new Map(),
```

Comment:
All of the SWRConfigs in tests need to reset the cache provider. If they don't, we get inconsistent behavior. This is a lot of repeated code in a bunch of files but we can deal with that later.